tools/rbac-gen/pkg/convert/tests/duplicates.out.yaml

```diff
@@ -12,7 +12,18 @@ rules:
   - create
   - delete
   - get
+  - list
+  - patch
   - update
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - configmaps/status
```

Comment:
I think configmaps don't have status subresource.
Should we update to generate only resources which has status subresource(e.g. deployments...), or set as future work?
Both is ok for me. :smile: